modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -61,6 +60,20 @@ The countdown to the first run of the selected test begins.
 Specified test times correspond with your time zone.
 ====
 
+// Explanation and table with schedule interval options by location type
+
+include::_partials/afm-ref-schedule-interval.adoc[leveloffset=+1]
+
+== Scheduling Long-Running Tests
+
+Tests for a monitor that are run in public locations silently fail when the run time exceeds 120 seconds. If you need longer tests, the best solution is to use a private location, because you can run tests of long duration in private locations. 
```

Comment:
Suggestion: "tests of long duration" to "tests that have a long duration" or "tests with a long duration"

The "of long duration" sounded funny to me.